src/http/commands/help/presenter.rb

```diff
@@ -102,57 +97,54 @@ def render_html_list(data, skip_wrapper: false)
                     value = data.attribute_declarations
                   end
                   html << "<li>#{key}"
-                  html << "<ul>"
-                  html << render_html_list(value, skip_wrapper: true)
-                  html << "</ul>"
+                  html << render_html_list(value)
                   html << "</li>"
                 end
-                html << "</ul>" unless skip_wrapper
+                html << "</ul>"
               when Manifest::Array
-                html << "<ul>" unless skip_wrapper
+                html << "<ul>"
                 data.relevant_manifest.each_pair do |key, value|
                   next if key == :element_type_declaration
 
                   if key.to_s == "type"
                     value = root_manifest.lookup_path(key, value)
                   end
                   html << "<li>#{key}"
-                  html << "<ul>"
-                  html << render_html_list(value, skip_wrapper: true)
-                  html << "</ul>"
+                  html << render_html_list(value)
                   html << "</li>"
                 end
-                html << render_html_list({ element_type: data.element_type }, skip_wrapper: true)
-                html << "</ul>" unless skip_wrapper
+                html << render_html_list({ element_type: data.element_type })
+                html << "</ul>"
               when Manifest::TypeDeclaration
                 manifest = data.relevant_manifest
-
+                html << "<ul>"
                 if manifest.is_a?(::Symbol)
                   html << "<li>"
                   html << foobara_reference_link(data.to_type)
                   html << "</li>"
                 else
-                  html << "<ul>" unless skip_wrapper
                   data.relevant_manifest.each_pair do |key, value|
                     if key.to_s == "type"
                       value = root_manifest.lookup_path(key, value)
                     end
                     html << "<li>#{key}"
-                    html << "<ul>"
-                    html << render_html_list(value, skip_wrapper: true)
-                    html << "</ul>"
+                    html << render_html_list(value)
                     html << "</li>"
                   end
-                  html << "</ul>" unless skip_wrapper
                 end
+                html << "</ul>"
               when Manifest::Type, Manifest::Command, Manifest::Error
+                html << "<ul>"
                 html << "<li>"
                 html << foobara_reference_link(data)
                 html << "</li>"
+                html << "</ul>"
```

Comment:
Since these are not collections, I think we should not render `<ul>` here. Maybe this is only give the right result since `::Array` is missing its `<ul>`s and `<li>`s.